core/src/test/java/hudson/model/JDKTest.java

```diff
@@ -0,0 +1,32 @@
+package hudson.model;
+
+import org.junit.Test;
```

Comment:
Please use JUnit 5 for tests instead of JUnit 4.
```suggestion
import org.junit.jupiter.api.Test;
```